mopidy/local/commands.py

```diff
@@ -76,6 +76,9 @@ def run(self, args, config):
             bytes(file_ext.lower()) for file_ext in excluded_file_extensions)
 
         library = _get_library(args, config)
+        if library == 1:
+            logger.error('Unable to scan local library: does not exist')
```

Comment:
I think it is better to:
- [ ] increase the level of the warning in `_get_library()` from warning to error
- [ ] remove the additional logging here
- [ ] return `None` instead of `1` from `_get_library()` on error
- [ ] check for `if library is None` here